wandb/sdk/lib/apikey.py

```diff
@@ -114,6 +116,14 @@ def prompt_api_key(  # noqa: C901
         key = browser_callback() if browser_callback else None
 
         if not key:
+            if not (settings.is_local or local):
+                host = app_url
+                for prefix in "http://", "https://":
+                    if app_url.startswith(prefix):
+                        host = app_url[len(prefix) :]
+                wandb.termlog(
+                    f"Logging into {host}. (Learn about deployment options: {wburls.get('deployments')})"
```

Comment:
If we really want to improve "local" W&B Server then maybe we say something like:

"Learn how to deploy a W&B Server locally: "https://wandb.me/wandb-server"